src/main/java/com/playground/playground/FeaturesHiddenLayersController.java

```diff
@@ -0,0 +1,196 @@
+package com.playground.playground;
+
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.GridPane;
+import javafx.scene.text.Text;
+
+
+import java.net.URL;
+import java.util.ResourceBundle;
+
+public class FeaturesHiddenLayersController implements Initializable {
+    @FXML
+    private Text numHiddenLayers;
+    @FXML
+    private Button addLayer;
+    @FXML
+    private Button removeLayer;
+    @FXML
+    private Button x1pow2button;
+    @FXML
+    private Button x2pow2button;
+    @FXML
+    private Button x1x2button;
+    @FXML
+    private Button sinx1button;
+    @FXML
+    private Button sinx2button;
+    @FXML
+    private Button add1;
+    @FXML
+    private Button remove1;
+    @FXML
+    private Button add2;
+    @FXML
+    private Button remove2;
+    @FXML
+    private Button add3;
+    @FXML
+    private Button remove3;
+    @FXML
+    private Button add4;
+    @FXML
+    private Button remove4;
+    @FXML
+    private Button add5;
+    @FXML
+    private Button remove5;
+    @FXML
+    private Button add6;
+    @FXML
+    private Button remove6;
+    @FXML
+    public GridPane neuralConnections;
+    private Button[] addButtons;
+    private Button[] removeButtons;
+    private Button[][] aButtons;
+    private int[] aButtonCounts;
+    private int i = 0;
+
+    @FXML
+    private Label neurons1, neurons2, neurons3, neurons4, neurons5, neurons6;
+
+    private Label[] neuronLabels;
+
+    /**
+     * Initializer for FeaturesHiddenLayersController.java
+     * @param location
+     * The location used to resolve relative paths for the root object, or
+     * {@code null} if the location is not known.
+     *
+     * @param resources
+     * The resources used to localize the root object, or {@code null} if
+     * the root object was not localized.
+     */
+    public void initialize(URL location, ResourceBundle resources) {
+        setButtonWithImage(x1pow2button, "playground-images/x1pow2button.jpg");
```

Comment:
I would strictly say no PNGs. We shouldn't do PNGs at all, I would always recommend in decreasing order of recommendation:

1. Use icons from some icons library or native icons (almost always or always do this), if you have a new icon (which we don't) import SVGs to these icons
2. Use .svg directly or .webp
3. Use some other vector graphics forms like .eps
4. .apng or .avif but only for lossless animation sequences
5. .png
6. .jpeg (please never do this)